deploy/osd-rebalance-infra-nodes/non-fr-sts-wif-privatelink/config.yaml

```diff
@@ -4,6 +4,9 @@ selectorSyncSet:
   - key: api.openshift.com/sts
     operator: NotIn
     values: ["true"]
+  - key: api.openshift.com/wif
+    operator: NotIn
+    values: ["true"]
```

Comment:
Could we also update the README in deploy/osd-rebalance-infra-nodes to match the new behavior?